README.md

```diff
@@ -21,5 +21,10 @@ A summary of Reflex functions is available in the [quick reference](Quickref.md)
 
 ### Hacking
 
-Use the `./scripts/hack-on reflex` script in [Reflex Platform](https://github.com/reflex-frp/reflex-platform) to checkout the source code of `reflex` locally in `reflex-platform/reflex` directory.
-Then do modifications to the source in place, and use the `./try-reflex` or `./scripts/work-on` scripts to create the shell to test your changes.
+Use the `./scripts/hack-on
+haskell-overlays/reflex-packages/dep/reflex` script in [Reflex
```

Comment:
Hmm, this instruction, though it may be accurate, has become a little awkwardly worded. Perhaps we should say something like "From the reflex-platform root directory, use the `./scripts/hack-on haskell-overlays/reflex-packages/dep/reflex` command to checkout the source code of reflex in `haskell-overlays/reflex-packages/dep/reflex`.

The directory in which hack-on will check out reflex will always be whatever directory you pass to it -- the one containing the thunk initially, so we should make sure that matches regardless.

This makes me wonder if reflex really should be buried so deep in reflex-platform. It makes sense to me for some of the thunks for core dependencies of reflex-platform perhaps to be slightly higher up in the directory structure -- maybe under a dep directory or something.

Reply:
I've pushed a rewording. Please take a look.